src/TestRunner.js

```diff
@@ -11,10 +11,11 @@
 //              tests.
 export default class TestRunner {
 
-  constructor(component, testSuites, startDelay, sendReport) {
+  constructor(component, testSuites, startDelay, reporter, sendReport) {
```

Comment:
Are we OK with how long this is getting?

Reply:
@jalada - would you prefer an options object?